bootstrapvz/common/tasks/grub.py

```diff
@@ -61,10 +99,14 @@ def run(cls, info):
 				device_map.write('(hd0) {device_path}\n'.format(device_path=device_path))
 				if not isinstance(p_map, partitionmaps.none.NoPartitions):
 					for idx, partition in enumerate(info.volume.partition_map.partitions):
-						device_map.write('(hd0,{prefix}{idx}) {device_path}\n'
-						                 .format(device_path=partition.device_path,
-						                         prefix=partition_prefix,
-						                         idx=idx + 1))
+						device_map.write(
+							'(hd0,{prefix}{idx}) {device_path}\n'
+							.format(
+								device_path=partition.device_path,
+								prefix=partition_prefix,
+								idx=idx + 1
+							)
+						)
```

Comment:
Looks like you are putting tabs (which should be used for indentation) where it should be spaces (used for alignment). I know this can be a pain some times (specially if your editor doesn't help), but this has been done in the entire code tree and doesn't looks like it will be changing soon.

This part of the code doesn't even had to be changed. I'd suggest you to drop this hunk.